src/sentry/static/sentry/app/api.jsx

```diff
@@ -2,7 +2,7 @@ import $ from 'jquery';
 import GroupActions from './actions/groupActions';
 import TeamActions from './actions/teamActions';
 
-class Request {
+export class Request {
```

Comment:
Is there a reason this is getting exported? I can't see it being imported anywhere.